benchmarks/infrastructure/aws/terraform/README.md

```diff
@@ -1,4 +1,11 @@
-# Create infrastructure with Terraform
+# AWS benchmarks
+
+The document describes step by step how to run Delta Lake benchmarks on AWS infrastructure.
+Currently, you can run benchmarks on either EMR on EKS.
+
+## Prerequisites
```

Comment:
I think a big question here thats not obvious to me is how do you configure what version of spark will be installed?

Reply:
For EKS I can add a "spark-version" parameter, so that a proper spark image will be used.
For EMR I can add yet another terraform variable for EMR version, so the user can indirectly choose which spark version to use.

Of course I'll describe it in README. Is that ok?